pytest_fixtures/component/templatesync.py

```diff
@@ -36,29 +34,56 @@ def create_import_export_local_dir(default_sat):
 @pytest.fixture(scope='session')
 def git_port(default_sat):
     """Allow port for git service"""
-    port = urlparse(settings.git.url).port
-    if port:
-        default_sat.execute(f'semanage port -a -t http_port_t -p tcp {port}')
+    default_sat.execute(f'semanage port -a -t http_port_t -p tcp {settings.git.http_port}')
+    default_sat.execute(f'semanage port -a -t ssh_port_t -p tcp {settings.git.ssh_port}')
 
 
 @pytest.fixture(scope='session')
-def git_repository(git_port):
+def git_pub_key(default_sat, git_port):
```

Comment:
This is a good `core` fixture, move this to `core/sys.py` !

Reply:
@jyejare Do you see any other current or future tests that might use this fixture?
This fixture is tied to Gitea/Github service. I see no other place in Satellite where git is used. Do you see any?